### PR TITLE
{compiler}[GCCcore/9.3.0] almosthere v1.0.10, limix-bgen v3.0.3

### PR DESCRIPTION
**Pull request by mboisson**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 21 out of 21 (2 easyconfigs in this PR)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.5
See https://gist.github.com/c91ba431519d2f4b32f818d1015609fb for a full test report.

**Comment by verdurin** [comment]
@mboisson could you upload a test report?

**Comment by verdurin** [comment]
Thanks @zao 

@mboisson here's how you do it: `eb --from-pr=<PR number> --debug --robot --upload-test-report`, the latter being the key one.

**Comment by mboisson** [comment]
Test report by @mboisson
**SUCCESS**
Build succeeded for 0 out of 0 (2 easyconfigs in this PR)
build-node.computecanada.ca - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.7.7
See https://gist.github.com/7c888f6010b777930d2edf9957e0c9a8 for a full test report.

**Comment by mboisson** [comment]
Test report by @mboisson
**FAILED**
Build succeeded for 1 out of 3 (2 easyconfigs in this PR)
build-node.computecanada.ca - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.7.7
See https://gist.github.com/8c78ecb69afe953033f6361dacba1f1d for a full test report.

**Comment by mboisson** [comment]
Yeah, that won't work.... `binutils` recipes don't work on our system, and we filter them out, but the test report seems to want to include them...

**Comment by verdurin** [comment]
Thanks anyway @mboisson - I can go with the build from @zao

File: easybuild/easyconfigs/a/almosthere/almosthere-1.0.10-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+checksums = ['36e943f6e76fba6d2638972aff71d767a5ba0e72c472acf8282729ef1b0a0a0d']
+configopts = '-DBUILD_SHARED_LIBS=ON'
+builddependencies = [
+    ('binutils', '2.34'),
+    ('CMake', '3.16.4'),
+]
+
+
+sanity_check_paths = {
+    'files': ['lib/libathr.so', 'include/athr.h'],
```
Comment (verdurin) [inline easybuild/easyconfigs/a/almosthere/almosthere-1.0.10-GCCcore-9.3.0.eb:29]: @mboisson Please use `SHLIB_EXT`, see e.g. https://github.com/easybuilders/easybuild-easyconfigs/blob/9c43fb52e7a393b54b7a98255d2f17e537eab6f9/easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-12.4-GCCcore-9.3.0.eb#L46

File: easybuild/easyconfigs/l/limix-bgen/limix-bgen-3.0.3-GCCcore-9.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    ('CMake', '3.16.4'),
+]
+dependencies = [
+    ('almosthere', '1.0.10'),
+    ('zstd', '1.4.4')
+]
+
+
+sanity_check_paths = {
+    'files': ['lib/libbgen.so', 'include/bgen.h'],
```
Comment (verdurin) [inline easybuild/easyconfigs/l/limix-bgen/limix-bgen-3.0.3-GCCcore-9.3.0.eb:29]: @mboisson same as above, please use `SHLIB_EXT`.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/limix-bgen/limix-bgen-3.0.3-GCCcore-9.3.0.eb (hunk 2 above)
Comment (mboisson) [reply easybuild/easyconfigs/l/limix-bgen/limix-bgen-3.0.3-GCCcore-9.3.0.eb]: done

File: easybuild/easyconfigs/a/almosthere/almosthere-1.0.10-GCCcore-9.3.0.eb (hunk 1 above)
Comment (mboisson) [reply easybuild/easyconfigs/a/almosthere/almosthere-1.0.10-GCCcore-9.3.0.eb]: done

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/91a674acf4d820e1448d6f5b96ed9f27 for a full test report.

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
Going in, thanks @mboisson!

**Pull request merged**